src/libraries/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Directory.Changed.cs

```diff
@@ -28,11 +31,11 @@ public void FileSystemWatcher_Directory_Changed_WatchedFolder()
         {
             using (var testDirectory = new TempDirectory(GetTestFilePath()))
             using (var dir = new TempDirectory(Path.Combine(testDirectory.Path, "dir")))
-            using (var watcher = new FileSystemWatcher(dir.Path, "*"))
+            using (var watcher = new FileSystemWatcher(GetDirectoryOrSymLink(dir.Path), "*"))
             {
                 Action action = () => Directory.SetLastWriteTime(dir.Path, DateTime.Now + TimeSpan.FromSeconds(10));
```

Comment:
Not related to your change, but I'm having trouble understanding why this test is not expecting any events after changing the last write time.

Reply:
The FSW in this case is watching the same folder whose `LastWriteTime` is being changed and this test demonstrates that changed to the watched directory are not being caught.

Reply:
On a second thought, I think these tests that use `ExpectEvent(watcher, 0, ...)` are wrong and instead should use `ExpectNoEvent`.